### DISCUSSION TITLE
It does not take effect after installation

### DISCUSSION THREAD

**Discussion by 911487458** [d8ahazard/sd_dreambooth_extension#1087] [Q&A]
model_cfg: DreamboothConfig = Body(description="The config to save"),

**Comment by G-force78** [comment]
Same, went through the lengthy collab install and the extension doesnt show up..

**Comment by leresteux** [comment]
hi, 
i got the same error :/

no new "dreambooth"  tab

---

Error executing callback app_started_callback for D:\Stable Diffusion\stable-diffusion-webui\extensions\sd_dreambooth_extension\scripts\api.py
Traceback (most recent call last):
  File "D:\Stable Diffusion\stable-diffusion-webui\modules\script_callbacks.py", line 107, in app_started_callback
    c.callback(demo, app)
  File "D:\Stable Diffusion\stable-diffusion-webui\extensions\sd_dreambooth_extension\scripts\api.py", line 524, in dreambooth_api
    model_cfg: DreamboothConfig = Body(description="The config to save"),
NameError: name 'DreamboothConfig' is not defined

what is my mistake ?

thx for the help

---
W11
i5
12 giga ram
RTX3060
---
![image](https://user-images.githubusercontent.com/21321409/226646757-6259c414-b2be-4187-911b-0c01d88ef96d.png)

**Comment by CarringtonCreative** [comment]
For anyone battling with this issue I `cd` into `stable-diffusion-webui\extensions\sd_dreambooth_extension` directory and ran `pip install -r requirements.txt` directly. Then I ran `pip install -r requirements.txt` in the `stable-diffusion-webui` directory. After that I reran the webui and I can now see the Dreambooth tab.

**Reply by neuralnetjerk** [reply]
BUT, does it train models?

**Comment by Lecho303** [comment]
hi，i cannot find 'Performance Wizard' in the Input column,actually i have not found the 'input column',what can i do?

**Comment by 192abc123** [comment]
I also run pip install -r requirements.txt, but it still doesn't show, it has been repeated for many times but it still doesn't work
